pkg/machines/components/vm/disks/diskAdd.jsx

```diff
@@ -234,6 +241,31 @@ const UseExistingDisk = ({ idPrefix, onValueChanged, validationFailed, dialogVal
     );
 };
 
+const CustomPath = ({ idPrefix, onValueChanged, dialogValues }) => {
+    return (<>
+        <FormGroup id={`${idPrefix}-file`}
+                   fieldId={`${idPrefix}-file-autocomplete`}
+                   label={_("Custom path")}>
+            <FileAutoComplete id={`${idPrefix}-file-autocomplete`}
+                placeholder={_("Path to file on host's file system")}
+                onChange={value => onValueChanged("file", value)}
+                superuser="try" />
+        </FormGroup>
+        <FormGroup id={`${idPrefix}-device`}
+                   fieldId={`${idPrefix}-select-device`}
+                   label={_("Device")}>
+            <FormSelect id={`${idPrefix}-select-device`}
+                        onChange={value => onValueChanged('device', value)}
+                        value={dialogValues.device}>
+                <FormSelectOption value="disk" key="disk"
+                                  label={_("Hard disk image")} />
+                <FormSelectOption value="cdrom" key="cdrom"
+                                  label={_("CD/DVD disc")} />
```

Comment:
s/disc/disk

Reply:
Actually disc is correct. It seems that both varians are correct, but "disc" is now used mostly associated with optical disc (so CD and DVDs and bluray): https://en.wikipedia.org/wiki/Spelling_of_disc
You can find the spelling used in relation to DVD: https://en.wikipedia.org/wiki/DVD
Even the word CD is a short of "compact disc": https://en.wikipedia.org/wiki/Compact_disc

Reply:
On you got me here. However CD and DVD both have the disc in the names already, compact disc (CD) for as mention above in the link.
So adding a second disc string sounds like repeating ourselves.

I am more leaning CD/DVD device.

And the option one line above could be just 'Disk device'. What do you think?

Reply:
I would shorten it to just "CD/DVD", but perhaps keep the "Hard disk" (get rid of image), since there are other types of disk devices, like floppy disk which libvirt allows too.

Reply:
actually now that I re-read your comments I am happy with the working as is